### ISSUE TITLE
Vimeo

### ISSUE THREAD

**Issue by chozze** [OriginalEXE/vidim#4]
It would be perfect if it supports also Vimeo!:)

**Comment by OriginalEXE** [comment]
Hello,

I can look into it. Are you aware that Vimeo Plus or PRO accounts are required? Unlike YouTube, Vimeo doesn't support full hiding of controls on free accounts.

**Comment by chozze** [comment]
Hmm, that sucks. If it is a case, then I think it is quite useless as not many people have PRO Vimeo account
But are you sure of that? https://github.com/pupunzi/jquery.mb.vimeo_player - there is an option to hide controls on pupunzi`s jquery player.

**Comment by OriginalEXE** [comment]
The way that library hides the controls is by cutting off the bottom part of the video where the controls are displayed, which is not ideal, though might be good enough for most people?

**Comment by chozze** [comment]
Yes I believe so, I can imagine many cases where the target DOM element is not 16:9 anyway 👍

**Comment by OriginalEXE** [comment]
Ok, I'll look into adding support for it then in the next few days :)

**Comment by chozze** [comment]
Hello mate, any luck with the Vimeo update?;)

**Comment by OriginalEXE** [comment]
Hi, apologies for the delay. I was getting married so had zero time to work on my side projects. WIll try to squeeze this in as soon as possible.
